src/components/ha-tabs.js

```diff
@@ -0,0 +1,57 @@
+import "@polymer/paper-tabs";
+
+const PaperTabsClass = customElements.get("paper-tabs");
+let subTemplate;
+
+class HaTabs extends PaperTabsClass {
+  static get template() {
+    if (!subTemplate) {
+      subTemplate = PaperTabsClass.template.cloneNode(true);
+
+      const superStyle = subTemplate.content.querySelector("style");
+
+      superStyle.appendChild(
+        document.createTextNode(`
+          .hidden, .not-visible {
+            display: none;
+          }
+        `)
+      );
+    }
+    return subTemplate;
+  }
+
+  ready() {
+    super.ready();
+    this.setScrollDirection("y", this.$.tabsContainer);
+
+    this.addEventListener("mousewheel", (e) => {
+      if (e.wheelDelta > 0) {
+        this._scrollToRight();
+      } else if (e.wheelDelta < 0) {
+        this._scrollToLeft();
+      } else {
+        return;
+      }
+      e.preventDefault();
+    });
+  }
+
+  _affectScroll(dx) {
+    super._affectScroll(dx);
+    this.$.tabsContainer.scrollLeft += dx;
```

Comment:
We do this twice now, is that intentional?

Reply:
Not intentional